calculation/calculation.go

```diff
@@ -35,17 +35,20 @@ func CalculateTotalDelegations(
 	}
 
 	for _, position := range positions {
-		totalDelegationAsset := position.Value.Assets[program.StakedAsset]
+		totalDelegationAsset := position.Value.AssetAmount(program.StakedAsset)
 
 		// Add in the value of LP tokens, according to the ratio of the pools at the snapshot
-		for assetId, amt := range position.Value.Assets {
+
+		for id, _ := range position.Value {
```

Comment:
I think the reason this wasn't uncovered in the unit tests is because we were only testing with asset IDs that *only* had a policy ID and no asset name.